js/index.js

```diff
@@ -132,3 +130,4 @@ document.getElementById('divBtn4').addEventListener('click', function () {
 });
 document.getElementById('hero-lookdown').addEventListener('click', goDown);
 
+
```

Comment:
please delete this line.

Reply:
this line called the event to scroll down, it's necesary

Reply:
She refers to the empty space, line 133

Reply:
done